pkgs/development/libraries/gobject-introspection/absolute_shlib_path.patch

```diff
@@ -1,6 +1,103 @@
---- ./giscanner/utils.py.orig	2014-08-14 22:05:05.055334080 +0200
-+++ ./giscanner/utils.py	2014-08-14 22:05:24.687497334 +0200
-@@ -110,17 +110,11 @@
+diff --git a/giscanner/scannermain.py b/giscanner/scannermain.py
+index 89ec193..8ee92c3 100755
+--- a/giscanner/scannermain.py
++++ b/giscanner/scannermain.py
+@@ -94,6 +94,34 @@ def get_windows_option_group(parser):
+     return group
+ 
+ 
++def _get_default_fallback_libpath():
++    outputs = os.environ.get("outputs", "out").split()
++    if "lib" in outputs:
```

Comment:
For multi output I suggest first checking for `$outputLib`. If empty use `$out`.